receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -34,6 +36,12 @@ type MetricFamily interface {
 	ToMetric() *metricspb.Metric
 }
 
+var (
+	k8SKeyLocation    = "k8s_location"
+	k8SKeyClusterName = "k8s_cluster_name"
```

Comment:
Why not just use resourcekeys.CloudKeyZone, K8SKeyClusterName, HostKeyName? (here and in the config)

Reply:
I came up with the new constants so that it would be clear what was going on when reading the config but, it probably makes the more sense to use the existing names to avoid another level of confusion.